### ISSUE TITLE
Several Questions about Paper

### ISSUE THREAD

**Issue by FerryHuang** [Kwai-Klear/RLEP#5]
Thanks for open-sourcing the paper and codes, it helps a lot.
I have two questions about the experiment:
1. For Table 1, what is the mini-bsz of full-version DAPO here and have you tried making the mini-bsz larger, like 128 for 4 updates per global step or even 512 for on-policy RL?
2. RLEP needs two-stage RL training which means two separate experiments. Have you tried an efficient alternative like mixing the earlier successful rollout trajectories of the question in the later epochs?

**Comment by zhhongzhi** [comment]
Thanks for your interest in our work!
1. For Table 1, the mini-batch size (mini-bsz) of the full DAPO setup is 32 (see `recipe/rlep/official_dapo.sh`). Fully on-policy training with a batch size of 512 performed poorly, mainly due to slow convergence (to my memory). We haven’t tested the configure like 128 yet because 3×-rollout training is resource-intensive; if we run this configuration later, we’ll append the result.
2. Yes—we tried the mixed-trajectory configuration. It was suboptimal. The core issue is that mixing trajectories collected at different training steps introduces instability (e.g., NaNs in the gradient norm). There are still issues to resolve before this approach is reliable.

**Comment by Antidotec** [comment]
Thanks for your work! Just to clarify, is DAPO-nodyn-bs64 the baseline for the RLEP training process? 
<img width="50%" alt="Image" src="https://github.com/user-attachments/assets/6458021e-ab88-4602-a85d-351b8f1c3a7e" />

**Comment by zhhongzhi** [comment]
> Thanks for your work! Just to clarify, is DAPO-nodyn-bs64 the baseline for the RLEP training process? <img alt="Image" width="50%" src="https://private-user-images.githubusercontent.com/49054251/477146987-6458021e-ab88-4602-a85d-351b8f1c3a7e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTUxNDI3NTIsIm5iZiI6MTc1NTE0MjQ1MiwicGF0aCI6Ii80OTA1NDI1MS80NzcxNDY5ODctNjQ1ODAyMWUtYWI4OC00NjAyLWE4NWQtMzUxYjhmMWMzYTdlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA4MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwODE0VDAzMzQxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEwZGMwYTkxMjYwODU5ZWExOTJiNjhiYjI0NmQ4MjYwOTMzNmFlZTAwOGVlYThkZDE4YTIwODdmMzdiMDJhOGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.FpaxnSoEM_PV39RnLkCFB7KA_0IseB5y3gEzCD2MJ8A">

Yeah.
